test/index.ts

```diff
@@ -14,11 +16,27 @@ function getSum(distributions: Reward[]): BigNumber {
 }
 
 describe('Script: rewards distribution ', () => {
-  it('Sum distributed to voters should be equal to initial amount set to distribute', async () => {
-    const TOTAL_REWARDS_AMOUNT = ethers.utils.parseEther('100');
-    const rewardsObject = await calculateRewardsAmounts(TOTAL_REWARDS_AMOUNT);
-    const rewardsSum = getSum(rewardsObject);
+  if (process.env.ETHEREUM_NETWORK === 'mainnet') {
+    it('Sum distributed to voters should be equal to initial amount set to distribute', async () => {
+      const TOTAL_REWARDS_AMOUNT = ethers.utils.parseEther('100');
+      const rewardsObject = await calculateRewardsAmounts(TOTAL_REWARDS_AMOUNT);
+      const rewardsSum = getSum(rewardsObject);
+      expect(Number(rewardsSum)).closeTo(Number(TOTAL_REWARDS_AMOUNT), 1000000);
+    });
+  } else {
+    it('Reward per voter should be distributed as per relative weight of their Sarco-VR balance', async () => {
```

Comment:
Yep this is what we want to keep. Thanks for this. 

There just might be other edge case tests we could add, but I guess the whole functionality is straightforward and intuitive enough that we should be good for now.

Or, well, there's also the argument that you gotta be super thorough when dealing with people's funds.

Thoughts? @sethhrbek @katien @knoll3 

I personally can't think of anything else to test, apart from maybe testing for large number of accounts?